Description: Update Makefile.global.in for package build.
 Set BINDIR to /usr/lib/mbsebbs/bin.
 Set ETCDIR to /etc/mbsebbs.
 Set DOCDIR to /usr/share/doc/mbsebbs.
 Set VARDIR to /var/lib/mbsebbs.
 Set INTDIR to /var/lib/mbsebbs/int.
 .
 This is for the Debian package build.
Author: Robert James Clay <jame@rocasa.us>
Last-Updated: 20130402

--- a/Makefile.global.in
+++ b/Makefile.global.in
@@ -10,11 +10,11 @@
 # for make install, where to put binaries and what owner to set
 SUBDIRS		= @SUBDIRS@
 PREFIX		= @prefix@
-BINDIR		= @prefix@/bin
-ETCDIR		= @prefix@/etc
-DOCDIR		= @prefix@/doc
-VARDIR		= @prefix@/var
-INTDIR		= @prefix@/share/int
+BINDIR		= @prefix@/usr/lib/mbsebbs/bin
+ETCDIR		= @prefix@/etc/mbsebbs
+DOCDIR		= @prefix@/usr/share/doc/mbsebbs
+VARDIR		= @prefix@/var/lib/mbsebbs
+INTDIR		= @prefix@/var/lib/mbsebbs/int
 OWNER		= @OWNER@
 GROUP		= @GROUP@
 ROWNER		= @ROWNER@
